src/Symfony/Contracts/HttpClient/Test/TestHttpServer.php

```diff
@@ -18,13 +18,8 @@ class TestHttpServer
 {
     private static array $process = [];
 
-    /**
-     * @param string|null $workingDirectory
-     */
-    public static function start(int $port = 8057/* , string $workingDirectory = null */): Process
+    public static function start(int $port = 8057, string $workingDirectory = __DIR__.'/Fixtures/web'): Process
```

Comment:
I think we should still accept null and have it be the default

Reply:
I have pushed an update. Is that what you had in mind?